osu.Game.Rulesets.Osu/Difficulty/Skills/OsuStrainSkill.cs

```diff
@@ -57,6 +59,20 @@ public override double DifficultyValue()
             return difficulty;
         }
 
+        public double CountTopWeightedSliders()
+        {
+            if (SliderStrains.Count == 0)
+                return 0;
+
+            double consistentTopStrain = DifficultyValue() / 10; // What would the top strain be if all strain values were identical
+
+            if (consistentTopStrain == 0)
+                return 0;
+
+            // Use a weighted sum of all strains. Constants are arbitrary and give nice values
+            return SliderStrains.Sum(s => 1.1 / (1 + Math.Exp(-10 * (s / consistentTopStrain - 0.88))));
```

Comment:
Use `DifficultyCalculationUtils.Logistic`

Reply:
We should change this for the other places too (separate PR)

Reply:
I'm pretty sure I did in most places